flutter/shell/platform/tizen/nui_autofill_popup.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/nui_autofill_popup.h"
+
+#include <dali-toolkit/dali-toolkit.h>
+#include <dali-toolkit/devel-api/controls/table-view/table-view.h>
+#include <dali-toolkit/public-api/controls/text-controls/text-label.h>
+
+#include "flutter/shell/platform/tizen/tizen_autofill.h"
+
+namespace flutter {
+bool NuiAutofillPopup::OnTouch(Dali::Actor actor,
+                               const Dali::TouchEvent& event) {
+  const Dali::PointState::Type state = event.GetState(0);
+  if (Dali::PointState::DOWN == state) {
+    auto t = actor.GetProperty(Dali::Actor::Property::NAME).Get<std::string>();
+    on_commit_(t);
+    OnHide();
+  }
+  return true;
+}
+
+void NuiAutofillPopup::OnHide() {
```

Comment:
I Think this seems to make more sense. 

```suggestion
void NuiAutofillPopup::OutsideTouched() {
```

Reply:
I think it would be better to distinguish between `OutsideTouched` and `Hide`. added `OutsideTouched` function that call `Hide`.